### PR TITLE
[BugFix] cos file system not found

### LINKED ISSUE #14473: [broker remove] broker load from COS,error
### Steps to reproduce the behavior (Required)
1. CREATE TABLE '...'
```
CREATE TABLE `aggregate_table_with_null` ( k0 int,v1 date,v2 datetime,v3 char(20),v4 varchar(20),v5 boolean,v6 int max,v7 int sum,v8 int min,v9 bigint max,v10 largeint min,v11 float sum,v12 double max,v13 decimal(27,9) max,v14 decimal32(9,5) sum,v15 decimal64(18,10) min,v16 decimal128(38,18) max,v17 string min ) ENGINE=OLAP aggregate KEY(k0, v1, v2, v3, v4, v5) COMMENT "OLAP" DISTRIBUTED BY HASH(k0) BUCKETS 3 PROPERTIES ( "replication_num" = "3", "storage_format" = "v2" );

remove broker name failed
LOAD LABEL test_broker_load_4f4212ac_717c_11ed_a027_00163e0e550b.label_50669a0e_717c_11ed_a027_00163e0e550b (DATA INFILE("cosn://dorisdb-test-1305337709/starrocks_test_data/system_data/hdfs_data/nose-ut-data/broker_data.csv") INTO TABLE `aggregate_table_with_null` COLUMNS TERMINATED BY "," FORMAT AS "csv" (k0, v1, v2, v3, v4, v5, v6, v7, v8, v9, v10, v11, v12, v13, v14, v15, v16, v17, v18, v19) )  WITH BROKER  ("fs.cosn.userinfo.secretId"="AKIDxxxxxxxxxxxx","fs.cosn.userinfo.secretKey"="8jAbxxxxxxx","fs.cosn.bucket.endpoint_suffix"="cos.ap-beijing.myqcloud.com") PROPERTIES("timeout"="300");
use broker name  sucess
LOAD LABEL test_broker_load_4f4212ac_717c_11ed_a027_00163e0e550b.label_50669a0e_717c_11ed_a027_00163e0e550b (DATA INFILE("cosn://dorisdb-test-1305337709/starrocks_test_data/system_data/hdfs_data/nose-ut-data/broker_data.csv") INTO TABLE `aggregate_table_with_null` COLUMNS TERMINATED BY "," FORMAT AS "csv" (k0, v1, v2, v3, v4, v5, v6, v7, v8, v9, v10, v11, v12, v13, v14, v15, v16, v17, v18, v19) )  WITH BROKER  hdfs_broker ("fs.cosn.userinfo.secretId"="AKIDxxxxxxxxxxxx","fs.cosn.userinfo.secretKey"="8jAbxxxxxxx","fs.cosn.bucket.endpoint_suffix"="cos.ap-beijing.myqcloud.com") PROPERTIES("timeout"="300");
```

### Expected behavior (Required)
load sucess
### Real behavior (Required)
image

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
mysql> select current_version();
+----------------------------+
| current_version()          |
+----------------------------+
| BRANCH-2.5-RELEASE 96a80a0 |
+----------------------------+
1 row in set (7.25 sec)

### PR DESCRIPTION
**Pull request by xiaoyong-z**
FE doesn't contain the tencent hadoop cos jar, we need to download it during the build of FE

Signed-off-by: xyz <a997647204@gmail.com>

## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #14473

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: build.sh [hunk 1]
```diff
@@ -336,6 +336,13 @@ if [ ${BUILD_FE} -eq 1 -o ${BUILD_SPARK_DPP} -eq 1 ]; then
         cp -r -p ${STARROCKS_HOME}/fe/spark-dpp/target/spark-dpp-*-jar-with-dependencies.jar ${STARROCKS_OUTPUT}/fe/spark-dpp/
         cp -r -p ${STARROCKS_THIRDPARTY}/installed/aliyun_oss_jars/* ${STARROCKS_OUTPUT}/fe/lib/
 
+        if [[ ! -f ${STARROCKS_OUTPUT}/broker_thirdparty_jars.tar.gz ]]; then
+            # download tencent hadoop cos jar, this can not be found in maven repository, so we download it manually
+            echo "download aliyun hadoop oss and tencent hadoop cos jar..."
+            wget http://cdn-thirdparty.starrocks.com/broker_thirdparty_jars.tar.gz -O ${STARROCKS_OUTPUT}/broker_thirdparty_jars.tar.gz
```
Comment (meegoo) [inline build.sh:342]: Can it be consistent with other third-party jars and integrated into docker image.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by chaoyli — APPROVED** [review]

**Review by murphyatwork — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**